apps/bfd-data-fda/Jenkinsfile

```diff
@@ -0,0 +1,138 @@
+                        awsAuth.assumeRole()
+
+                        withCredentials([string(credentialsId: 'bfd-aws-account-id', variable: 'ACCOUNT_ID')]) {
+                            codeArtifactAuthToken = sh(
+                            returnStdout: true,
+                            script: '''
+aws codeartifact get-authorization-token --domain bfd-mgmt \
+ --domain-owner "$ACCOUNT_ID" \
+ --output text --query authorizationToken
+'''
+                            ).trim()
+                            codeArtifactEndpoint = sh(
+                                returnStdout: true,
+                                script: '''
+aws codeartifact get-repository-endpoint \
+--domain bfd-mgmt --repository bfd-mgmt \
+--format maven --output text
+'''
+                            )
+                            env.CODEARTIFACT_AUTH_TOKEN = codeArtifactAuthToken
+                            env.CODEARTIFACT_ENDPOINT = codeArtifactEndpoint
+                        }
+                        sh '''
+cat <<EOF > ~/.m2/settings.xml
+<settings xmlns="http://maven.apache.org/settings/1.0.0" xmlns:xsi="http://www.w3.org/2001/xmlschema-instance"
+xsi:schemalocation="http://maven.apache.org/settings/1.0.0 https://maven.apache.org/xsd/settings-1.0.0.xsd">
+<servers>
+    <server>
+    <username>aws</username>
+    <password>${CODEARTIFACT_AUTH_TOKEN}</password>
+    <id>bfd-mgmt-bfd-mgmt</id>
+    </server>
+</servers>
+</settings>
+EOF
+'''
+                    }
+                }
+            }
+        }
+        stage('Maven Build') {
+            steps {
+                script {
+                    container('bfd-cbc-build') {
+                    // Calculate FDA DrugCode Sha256
+                        fdaDrugCodeSha256 = sh(
+                            returnStdout: true,
+                            script: '''
+curl -fsSL https://www.accessdata.fda.gov/cder/ndctext.zip | sha256sum | awk '{ print $1 }'
+'''
+                        )
+                        awsAuth.assumeRole()
+                        ssmLookupSha256 = sh(
+                            returnStdout: true,
+                            script: '''
+if parameter_lookup="$(aws ssm get-parameter --name /bfd/mgmt/versioning/nonsensitive/bfd-data-fda \
+--region us-east-1 --output text --query Parameter.Value 2>/dev/null)"; then
+   echo "$parameter_lookup"
+fi
+'''
+                        )
+
+                        if (ssmLookupSha256.isEmpty() || fdaDrugCodeSha256.trim() != ssmLookupSha256.trim()) {
+                            dir('apps/bfd-data-fda') {
+                                def quietFlags = params.verbose_mvn_logging ? '' : '--quiet'
+                                sh "mvn ${quietFlags} clean install"
+                                sh """
+mvn ${quietFlags} exec:java \
+-Dexec.mainClass="gov.cms.bfd.data.fda.utility.App" \
+-Dexec.args="./src/main/resources/"
+"""
+                                sh 'echo "$CODEARTIFACT_ENDPOINT"'
```

Comment:
This was a poor diagnostic we were using which can be removed safely now. Also harmless to keep in. I'll leave it up to you.
```suggestion
```

Reply:
done